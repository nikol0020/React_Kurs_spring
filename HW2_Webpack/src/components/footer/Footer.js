import React, {Component} from 'react';
import './Footer.scss';

const Footer = () => {
    return (
        <div className="footer">
            <h2 className='footer__text'>React Todo App !!!</h2>
        </div>
    );
};

export default Footer;
