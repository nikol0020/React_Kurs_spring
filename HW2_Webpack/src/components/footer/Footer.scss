.footer {
  width: 100%;
  height: 100px;
  background-color: #60f;
  position: absolute;
  bottom: 0;
}