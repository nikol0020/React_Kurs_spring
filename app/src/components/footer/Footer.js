import React, {Component} from 'react';
import './Footer.scss';

const Footer = () => {
    return (
        <div className="footer">
            <div className='footer__text'>React Todo App !!!</div>
        </div>
    );
};

export default Footer;
