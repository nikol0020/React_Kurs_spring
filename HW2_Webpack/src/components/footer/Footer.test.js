import React from 'react';
import Footer from './Footer';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render, configure } from 'enzyme';

describe('<Footer /> shallow rendering', () => {

    it('should render Footer', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render a div', () => {
        const wrapper = shallow(
            <div className="footer">
                <div className='footer__text'>React Todo App !!!</div>
            </div>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('<div>Something went wrong.</div>', () => {
        const wrapper = shallow(
            <div>React Todo App !!!</div>
        );
        expect(wrapper.find('div').text()).toBe('React Todo App !!!');
    });

});