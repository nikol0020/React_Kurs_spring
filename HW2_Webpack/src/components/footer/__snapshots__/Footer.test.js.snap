// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> shallow rendering should render Footer 1`] = `
<div
  className="footer"
>
  <div
    className="footer__text"
  >
    React Todo App !!!
  </div>
</div>
`;

exports[`<Footer /> shallow rendering should render a div 1`] = `
<div
  className="footer"
>
  <div
    className="footer__text"
  >
    React Todo App !!!
  </div>
</div>
`;
